src/cc/arduino/plugins/wifi101/UpdaterImpl.java

```diff
@@ -50,42 +64,80 @@ public UpdaterImpl() throws Exception {
 		});
 		Base.setIcon(this);
 
-		for (WINC1500Firmware firmware : WINC1500Firmware.available)
-			getFirmwareSelector().addItem(firmware);
+		fwAvailable.add(new WINCFlasher("WINC1501 Model B", "19.5.4", "firmwares/WINC1500/19.5.4/m2m_aio_3a0.bin", true, new ArrayList<String>(asList("Arduino/Genuino MKR1000"))));
+		fwAvailable.add(new WINCFlasher("WINC1501 Model B", "19.5.2", "firmwares/WINC1500/19.5.2/m2m_aio_3a0.bin", true, new ArrayList<String>(asList("Arduino/Genuino MKR1000"))));
+		fwAvailable.add(new WINCFlasher("WINC1501 Model B", "19.4.4", "firmwares/WINC1500/19.4.4/m2m_aio_3a0.bin", true, new ArrayList<String>(asList("Arduino/Genuino MKR1000"))));
+		fwAvailable.add(new WINCFlasher("WINC1501 Model A", "19.4.4", "firmwares/WINC1500/19.4.4/m2m_aio_2b0.bin", true, new ArrayList<String>(asList("Arduino WiFi 101 Shield"))));
+		fwAvailable.add(new NinaFlasher("NINA firmware", "1.0.0", "firmwares/NINA/1.0.0/nina-fw.bin", false, new ArrayList<String>(asList("Arduino MKR WiFi 1010", "Arduino MKR Vidor 4000", "Arduino UNO WiFi REV2"))));
+		fwAvailable.add(new NinaFlasher("NINA firmware", "1.1.0", "firmwares/NINA/1.1.0/NINA_W102.bin", true, new ArrayList<String>(asList("Arduino MKR WiFi 1010", "Arduino MKR Vidor 4000", "Arduino UNO WiFi REV2"))));
```

Comment:
We need to move the new "1.1.0" entry in the list to be first so it's the default.

Would be good to also rename the 1.0.0 file to `NINA_W102.bin` for consistency.